eu.html

```diff
@@ -170,13 +170,13 @@
 <p ><b><span>5.
 Who are we and how to contact us</span></b></p>
 
-<p ><b><span >5.1 Our company information.</span></b><span>&nbsp;TransferWise
+<p ><b><span >5.1 Our company information.</span></b><span>&nbsp;Wise

```

Comment:
Should say this

5.1 Our company information. Wise is the trading name of TransferWise Europe NV/SA, a company incorporated under the law of Belgium with company number 0713.629.988 ("Wise", "we", "us", or "our" as applicable).